generators/client/templates/angular/package.json.ejs

```diff
@@ -42,6 +42,9 @@
     "core-js": "2.5.3",
     "jquery": "3.3.1",
     "moment": "2.21.0",
+    <%_ if (!skipUserManagement && authenticationType !== 'oauth2') { _%>
+    "ngx-clipboard": "11.0.0",
```

Comment:
do we really need this, I would say copy feature is a luxury

Reply:
@agaspardcilia can we remove this and let user just copy manually? I think having this dependency just for this doesnt make sense